graph/iterator/mock_ts_test.go

```diff
@@ -35,7 +35,7 @@ func (qs *store) ValueOf(s string) graph.Value {
 	return nil
 }
 
-func (qs *store) ApplyDeltas([]graph.Delta) error { return nil }
+func (qs *store) ApplyDeltas([]graph.Delta, bool, bool) error { return nil }
```

Comment:
I'll take it. I might have done a struct for this case, but (at the moment) it's two bools, and may get updated. So, this may get upgraded to full options or to a struct if we add another flag, but for now, I'm willing to go with it.